actions/github-release-dependency/main.go

```diff
@@ -122,14 +125,27 @@ func main() {
 		for _, a := range r.Assets {
 			if g.MatchString(*a.Name) {
 				versions[*r.TagName] = *a.BrowserDownloadURL
+				sources[*r.TagName] = fmt.Sprintf("https://github.com/%s/%s/archive/refs/tags/%s.tar.gz", o, repo, originalTagName[*r.TagName])
```

Comment:
how is `originalTagName[*r.TagName]` different from `*r.TagName` ?

Reply:
I mentioned in the comment but this action overwrites the tag name with the normalised version, so I have to grab the original before this happens in order to build the source url using the proper tag name